runtimes/spiritnet/src/lib.rs

```diff
@@ -471,6 +471,18 @@ impl pallet_collective::Config<TechnicalCollective> for Runtime {
 	type WeightInfo = weights::pallet_collective::WeightInfo<Runtime>;
 }
 
+type TipperCollective = pallet_collective::Instance3;
+impl pallet_collective::Config<TipperCollective> for Runtime {
```

Comment:
I actually think, that we don't need the collective pallet here. It only adds voting power. Tippers don't need that.